### DISCUSSION TITLE
Pydeck, different tooltip per layer?

### DISCUSSION THREAD

**Discussion by PartiallyObservable** [visgl/deck.gl#6828] [Q&A]
I would like to have a custom tool tip for each layer rendered by Deck. Is this possible in pydeck?

This [discussion](https://github.com/visgl/deck.gl/discussions/6512) has replies that makes it seem like it is possible, but I am unsure on how it translates to pydeck syntax.

**Comment by ajduberstein** [comment]
Currently it's not possible, as the deck.gl [tooltip we use is for the Deck object and not for individual layers](https://github.com/visgl/deck.gl/blob/1049956d723fe7a92c39ea203bc2b5c5fe00cc36/modules/jupyter-widget/src/playground/widget-tooltip.js#L122).

It's possible to implement this in a future version of pydeck.

While you can't implement custom tooltips per layer just yet, you can set `pickable=False` on a specific `Layer` to prevent its info from appearing in your tooltip.
